app/packs/controllers/autosubmit_controller.js

```diff
@@ -22,6 +22,31 @@ export default class extends Controller {
     }, 0)
   }
 
+  capacitySubmit(event) {
+    clearTimeout(this.timeout)
+
+    this.timeout = setTimeout(() => {
+      var min_capacity = parseInt(this.min_capacityTarget.value)
+      var max_capacity = parseInt(this.max_capacityTarget.value)
+
+      if (min_capacity > max_capacity) {
+        this.capacity_errorTarget.classList.add("capacity-error--display")
+        this.capacity_errorTarget.classList.remove("capacity-error--hide")
+        this.capacity_errorTarget.innerText = "Min should be smaller than Max"
+        event.preventDefault()
```

Comment:
Which default behavior are we trying to stop here ?

Reply:
By default, the form will be submitted. I think we shouldn't submit a form if min capacity > max capacity. If submitted the search will return an empty result.

Reply:
Based on follow up discussion with Rita, this line of code is not required and was removed.